pages/api/hello.ts

```diff
@@ -1,13 +0,0 @@
-// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
I guess this file was autogenerated?

Reply:
Yep!! This file came with the project when it was first created! So I just removed it since it has no purpose in our project:)